### ISSUE TITLE
Using with Visual Effects for Google Meet and then disabling/uninstalling Visual Effects for Google Meet disables recognition of camera

### ISSUE THREAD

**Issue by d-g-omel** [FlyOrBoom/mercator#6]
Installed Visual Effects for Google Meet and then installed Mercator Studio (Chrome). Disabling Visual Effects disabled Mercator Studio's recognition of camera. Re-enabling Visual Effects re-enabled camera recognition for Mercator Studio. Uninstalling Visual Effects had same result. Problem persists with uninstall and reinstall of Mercator Studio. If Visual Effects for Google Meet is installed, Mercator Effects will only recognize camera with Visual Effects enabled.

**Comment by xingyzt** [comment]
So the two extensions are clashing?

**Comment by xingyzt** [comment]
I don't have access to the source code of Visual Effects for Google Meet. I'll reach out to its developer and see what we can do to make them interoperable.

**Comment by d-g-omel** [comment]
Yes, I cannot uninstall/disable visual effects without it disabling Mercator's camera recognition. I get the orange and yellow colored bars in the preview panes. I have not tried to replicate the problem using another machine or account. Reinstalling or re-enabling the Visual Effects extension re-enables Mercator's recognition of the camera. I also did not provide you with the version of Chrome I am using: Version 84.0.4147.125 (Official Build) (64-bit)

**Comment by xingyzt** [comment]
Mercator stops the video when it detects _something_ stopped using the stream. I think the problem now is differentiating who is triggering the inactive event, Visual Effects or Google Meet.
```js
			new_stream.addEventListener('inactive',() => {
				old_stream.getTracks().forEach(track => {
					track.stop()
				})
				context.clearRect(0,0,w,h)
				video.srcObject = null
			})
```
